src/logging.rs

```diff
@@ -9,6 +9,7 @@ use serde::Deserialize;
 pub struct LoggingConfig {
     /// Configures a Gelf Logger
     pub gelf: Option<GelfConfig>,
+    pub console: Option<ConsoleConfig>
```

Comment:
Instead of making it optional, could we give it a default value with `[serde(default='WARN')]`? This way we could get rid of the if during the initialization of the env logger.